README.md

```diff
@@ -84,14 +84,29 @@ Due to [network limitation](https://docs.docker.com/docker-for-mac/networking/#k
 $ EXPOSE_HOST_PORTS=true make docker-testacc
 ```
 
+Branching the Provider
+---------------------------
+
+The provider is branched into two release lines that have major version alignment with Rancher 2.6 and 2.7. The `release/v2` branch with 2.0.0+ is aligned with Rancher 2.6 and `master` with 3.0.0+ is aligned with Rancher 2.7. Terraform fixes and new features will be available on `master` but only bug fixes will be backported to `release/v2.`
+
+Aligning major provider releases with minor Rancher releases means,
+
+* We can follow semver 
+* We can cut patch/minor versions on an as-needed basis to fix bugs or add new resources 
+* We have 'out of band' flexibility and are only tied to releasing a new version of the provider when we get a new 2.x Rancher minor version.
+
+See the [compatibility matrix](https://github.com/rancher/terraform-provider-rancher2/tree/master/docs/compatibility-matrix.md) for details.
```

Comment:
Suggesting to use relative link (https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files) here as it's more resilient to changes to URLs and will work if we backport this docs change to another branch without needing to change it (unless we don't want to port actual `compatibility-matrix.md` file and always point to it in the master 🤷 ).